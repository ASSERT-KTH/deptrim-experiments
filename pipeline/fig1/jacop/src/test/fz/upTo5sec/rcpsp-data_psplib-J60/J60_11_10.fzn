predicate jacop_cumulative(array [int] of var int: s,array [int] of var int: d,array [int] of var int: r,var int: b);
array [1..2] of int: X_INTRODUCED_62 = [1,-1];
array [1..49] of int: X_INTRODUCED_119 = [7,7,9,8,7,9,6,10,6,7,9,4,9,3,10,1,3,6,1,9,10,7,2,3,7,5,6,8,6,4,2,4,9,9,4,1,2,9,1,10,8,7,10,10,9,3,3,6,8];
array [1..49] of int: X_INTRODUCED_120 = [1,5,7,9,3,9,10,6,4,4,7,10,2,7,6,3,2,4,2,2,1,4,1,2,8,7,9,8,6,4,5,6,1,8,1,5,10,5,3,5,3,2,3,6,7,3,5,9,4];
array [1..45] of int: X_INTRODUCED_122 = [5,6,5,9,10,8,9,7,7,7,8,1,7,3,2,2,1,1,10,4,4,3,9,2,6,2,8,6,9,2,2,10,6,2,8,8,9,9,8,9,7,7,6,8,4];
array [1..45] of int: X_INTRODUCED_123 = [1,5,7,9,6,4,10,7,6,4,4,1,2,4,2,2,1,1,1,2,8,8,6,10,5,6,1,8,1,5,8,10,5,3,5,6,10,3,2,9,7,3,5,9,4];
array [1..43] of int: X_INTRODUCED_125 = [8,4,6,1,4,7,3,4,7,1,8,5,2,6,8,7,5,10,8,5,2,10,7,9,10,9,8,4,1,9,8,2,10,7,6,2,4,3,1,10,2,1,9];
array [1..43] of int: X_INTRODUCED_126 = [1,1,5,7,3,9,4,4,7,10,7,6,4,1,2,2,1,4,1,1,2,8,7,6,9,10,5,6,8,1,8,5,3,6,10,2,9,3,6,7,3,5,4];
array [1..44] of int: X_INTRODUCED_128 = [10,7,4,4,3,6,6,1,3,10,10,4,7,1,2,8,5,8,1,1,6,6,1,1,2,6,10,4,6,6,1,9,1,3,1,1,9,6,3,9,2,4,6,1];
array [1..44] of int: X_INTRODUCED_129 = [1,1,5,7,9,9,10,4,4,7,10,7,4,4,1,3,4,2,4,2,8,7,9,8,6,4,9,10,1,8,1,8,10,5,5,6,10,3,2,6,3,5,9,4];
var 0..306: X_INTRODUCED_0;
var 0..306: X_INTRODUCED_1;
var 0..306: X_INTRODUCED_2;
var 0..306: X_INTRODUCED_3;
var 0..306: X_INTRODUCED_4;
var 0..306: X_INTRODUCED_5;
var 0..306: X_INTRODUCED_6;
var 0..306: X_INTRODUCED_7;
var 0..306: X_INTRODUCED_8;
var 0..306: X_INTRODUCED_9;
var 0..306: X_INTRODUCED_10;
var 0..306: X_INTRODUCED_11;
var 0..306: X_INTRODUCED_12;
var 0..306: X_INTRODUCED_13;
var 0..306: X_INTRODUCED_14;
var 0..306: X_INTRODUCED_15;
var 0..306: X_INTRODUCED_16;
var 0..306: X_INTRODUCED_17;
var 0..306: X_INTRODUCED_18;
var 0..306: X_INTRODUCED_19;
var 0..306: X_INTRODUCED_20;
var 0..306: X_INTRODUCED_21;
var 0..306: X_INTRODUCED_22;
var 0..306: X_INTRODUCED_23;
var 0..306: X_INTRODUCED_24;
var 0..306: X_INTRODUCED_25;
var 0..306: X_INTRODUCED_26;
var 0..306: X_INTRODUCED_27;
var 0..306: X_INTRODUCED_28;
var 0..306: X_INTRODUCED_29;
var 0..306: X_INTRODUCED_30;
var 0..306: X_INTRODUCED_31;
var 0..306: X_INTRODUCED_32;
var 0..306: X_INTRODUCED_33;
var 0..306: X_INTRODUCED_34;
var 0..306: X_INTRODUCED_35;
var 0..306: X_INTRODUCED_36;
var 0..306: X_INTRODUCED_37;
var 0..306: X_INTRODUCED_38;
var 0..306: X_INTRODUCED_39;
var 0..306: X_INTRODUCED_40;
var 0..306: X_INTRODUCED_41;
var 0..306: X_INTRODUCED_42;
var 0..306: X_INTRODUCED_43;
var 0..306: X_INTRODUCED_44;
var 0..306: X_INTRODUCED_45;
var 0..306: X_INTRODUCED_46;
var 0..306: X_INTRODUCED_47;
var 0..306: X_INTRODUCED_48;
var 0..306: X_INTRODUCED_49;
var 0..306: X_INTRODUCED_50;
var 0..306: X_INTRODUCED_51;
var 0..306: X_INTRODUCED_52;
var 0..306: X_INTRODUCED_53;
var 0..306: X_INTRODUCED_54;
var 0..306: X_INTRODUCED_55;
var 0..306: X_INTRODUCED_56;
var 0..306: X_INTRODUCED_57;
var 0..306: X_INTRODUCED_58;
var 0..306: X_INTRODUCED_59;
var 0..307: objective:: output_var;
array [1..60] of var int: s:: output_array([1..60]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..49] of var int: X_INTRODUCED_121 ::var_is_introduced  = [X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..45] of var int: X_INTRODUCED_124 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_6,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_12,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..43] of var int: X_INTRODUCED_127 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_20,X_INTRODUCED_22,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_33,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_43,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_59];
array [1..44] of var int: X_INTRODUCED_130 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_14,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_25,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_54,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
array [1..61] of var int: X_INTRODUCED_134 ::var_is_introduced  = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54,X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,objective];
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_7],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_14],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_3],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_4],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_6],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_5],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_9],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_51],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_8],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_16],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_22],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_40],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_56],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_11],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_15],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_20],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_17],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_10],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_21],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_38],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_13],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_56],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_12],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_33],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_29],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_35],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_28],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_43],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_52],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_18],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_27],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_37],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_53],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_49],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_23],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_31],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_19],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_24],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_45],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_25],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_26],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_47],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_28],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_45],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_32],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_34],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_38],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_48],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_39],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_30],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_49],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_26,X_INTRODUCED_44],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_30],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_42],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_54],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_41],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_39],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_53],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_40],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_51],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_31,X_INTRODUCED_36],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_40],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_46],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_47],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_50],-4);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_35,X_INTRODUCED_47],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_49],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_37,X_INTRODUCED_39],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_54],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_39,X_INTRODUCED_52],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_40,X_INTRODUCED_57],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_41,X_INTRODUCED_53],-1);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_42,X_INTRODUCED_43],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_44],-8);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_44,X_INTRODUCED_48],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_45,X_INTRODUCED_55],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_46,X_INTRODUCED_55],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_58],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_48,X_INTRODUCED_58],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_49,X_INTRODUCED_54],-10);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_50,X_INTRODUCED_56],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_51,X_INTRODUCED_59],-2);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_52,X_INTRODUCED_55],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_53,X_INTRODUCED_57],-3);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_54,X_INTRODUCED_57],-6);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_55,X_INTRODUCED_59],-7);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_56,X_INTRODUCED_59],-3);
constraint jacop_cumulative(X_INTRODUCED_121,X_INTRODUCED_120,X_INTRODUCED_119,56);
constraint jacop_cumulative(X_INTRODUCED_124,X_INTRODUCED_123,X_INTRODUCED_122,49);
constraint jacop_cumulative(X_INTRODUCED_127,X_INTRODUCED_126,X_INTRODUCED_125,39);
constraint jacop_cumulative(X_INTRODUCED_130,X_INTRODUCED_129,X_INTRODUCED_128,35);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_57,objective],-5);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_58,objective],-9);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_59,objective],-4);
solve :: int_search(X_INTRODUCED_134,smallest,indomain_min,complete) minimize objective;
