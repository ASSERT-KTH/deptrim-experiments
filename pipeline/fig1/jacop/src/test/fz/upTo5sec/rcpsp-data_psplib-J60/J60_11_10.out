objective = 58;
s = array1d(1..60, [0, 0, 0, 1, 1, 5, 1, 1, 8, 5, 11, 8, 15, 14, 1, 10, 8, 10, 16, 14, 8, 11, 10, 12, 15, 17, 17, 9, 25, 15, 21, 14, 16, 20, 13, 23, 26, 8, 15, 23, 28, 32, 10, 25, 33, 17, 33, 32, 43, 36, 17, 28, 27, 33, 46, 36, 20, 52, 49, 43]);
----------
==========
